group.go

```diff
@@ -37,6 +37,31 @@ type Groups struct {
 	Groups       []*Group       `json:"groups"`
 }
 
+// GetGroupOpts contains a payload for Group and Groups method
+//		embed:		Embeds nested resources that match the supplied path.
+// 					You can specify this parameter multiple times with different values
+type GetGroupOpts struct {
+	Embed []string `url:"embed,omitempty"`
+}
+
+// DeleteGroupOpts contains a payload for DeleteGroup method
+//		force:		Only one deployment can be applied to one application at the same time.
+//					If the existing deployment should be canceled by this change, you can set force=true
+//					Caution: setting force=true xill cancel the current deployment. This parameter should be used only,
```

Comment:
I'd opt for dropping the implication of what a force means. It's actually a bit [more complicated](https://github.com/mesosphere/marathon/issues/1346) than what your description says. (FWIW, in my organization we force deployments by default since we're not using groups).

My suggestion would be to just say (stolen from the REST docs):

`force:    overrides a currently running deployment.`